devIocStats/devIocStatsAnalog.c

```diff
@@ -286,6 +286,7 @@ static validGetParms statsGetParms[]={
 	{ "cbHighQueueHiWtrMrk",	statsCbHighQHiWtrMrk,	QUEUE_TYPE },
 	{ "cbHighQueueUsed",		statsCbHighQUsed,	QUEUE_TYPE },
 	{ "cbHighQueueOverruns",	statsCbHighQOverruns,	QUEUE_TYPE },
+  { "cpu_temperature", statsCpuTemperature, LOAD_TYPE },
```

Comment:
I know that these files are wildly inconsistent about their indentation, but it would be nice if the new code at least matched the local indentation as much as possible.

Reply:
My editor uses blanks for indentation, obviously the other lines were indented with tabs.
I changed the indentation to a tab now for this line